EventGenerator/src/PionFilter_module.cc

```diff
@@ -26,22 +26,26 @@ namespace mu2e {
       struct Config {
         using Name=fhicl::Name;
         using Comment=fhicl::Comment;
-        fhicl::Atom<int> diagLevel{Name("diagLevel"),0};
+        fhicl::Atom<int> diagLevel{Name("diagLevel"),1};
         fhicl::Atom<double> tmin{Name("tmin"),0};
         fhicl::Atom<double> tmax{Name("tmax"),1e6};
         fhicl::Atom<bool> isNull{Name("isNull"),true};
```

Comment:
Is it normal to have a filter module default to off even if the filtering parameters are configured?

Reply:
will fix